packages/version-select/util.ts

```diff
@@ -10,3 +10,19 @@ export function getVersionFromPath(path: string): string | undefined {
 
   return version
 }
+
+/**
+ * Removes a semver string from a path, and returns the new path.
+ * Returns the original string if no semver is present.
+ */
+export function removeVersionFromPath(path: string): string {
+  const pathSegments = path.split('/')
+
+  const i = pathSegments.findIndex((el) => SEMVER_REGEX.test(el))
```

Comment:
Will this regex match on something like `v0.5.x`? It doesn't look like it will... 🤔

Reply:
It will not match `v0.5.x`..._yet_

(I'm using this [semver regex](https://github.com/hashicorp/react-components/blob/main/packages/version-select/util.ts#L1) for now)

Reply:
Addressed in https://github.com/hashicorp/react-components/pull/342/commits/845bcfe9b141969a5fab2cb0cb75cc57b67e66d3